src/components/TopicSelector/TopicSelector.js

```diff
@@ -50,23 +69,28 @@ export default class TopicSelector extends Component {
 
   componentDidMount () {
     this.updateSelected()
+    this.props.fetchDefaultTopics({ groupSlug: get('forGroups[0].slug', this.props) })
```

Comment:
here it is

Reply:
Yes, ultimately I moved the fetching into `TopicsSelector` and out of `PostEditor`.